### ISSUE TITLE
"Ask" button not working

### ISSUE THREAD

**Issue by tpschmidt** [ai-genie/chatgpt-vscode#11]
### Verify it's not a duplicate bug report

- [X] I searched for similar bugs reported on GitHub and nobody has reported a similar issue. <a href="https://github.com/ai-genie/vscode-chatgpt/issues?q=" target="_blank">Search all issuees</a>.
- [X] I read <a href="https://github.com/ai-genie/vscode-chatgpt#troubleshooting" target="_blank">Troubleshooting guide on Readme</a> and there is not a guide on how to solve my problem.

### Describe the Bug

Installed the extension.
Opened the Genie tab to ask a question.
=> Button click does nothing.

When selecting a code block and invoking any Genie action, there's also no effect.

There are no console error messages or anything else.

- MacOS Monterey 12.6
- VSCode 1.76.2 (nothing customized, very few extensions)

![genie](https://user-images.githubusercontent.com/13495700/226826738-c938765b-e316-49b4-859c-6719970bc2ab.gif)

### Please tell us if you have customized any of the extension settings or whether you are using the defaults.

Using the defaults.

**Comment by dahelg** [comment]
Same problem. 😕

**Comment by genieai-info** [comment]
@dahelg are you also using Mac? The editor view uses `text-davinci-003` to make sure you are getting a code response that will be linted well in editor. Do you mind switching user model setting to `text-davinci-003` in your settings.json file and check if you are able to ask any questions in the `Conversation View`?

Could you also try using keyboard shortcut to send the prompt? <kbd>Ctrl</kbd>+<kbd>Enter<kbd>

Please check to see if you have any errors/logs printed out in the Output->Window in your VS Code.
![image](https://user-images.githubusercontent.com/128560534/227049832-9e3f9829-85a3-4e50-9f3e-06f1f13ff82d.png)

**Comment by dahelg** [comment]
@genieai-info Yes, I'm using MacOS Ventura Version 13.2.1. I've set the model to `text-davinci-003`, now. Before it has been `gpt-3.5-turbo`. Conversation view does not work anyway, hotkey `Ctrl`+`Enter`  is already bound to github copilot extension.

UPDATE:
After I've tried out a few more things, it works, now:
- I uninstalled another ChatGPT extension.
- Cleared open ai API key and set it again, then I reloaded vscode.
- Also I've checked my OpenAI Organization ID, but this has already been set correctly.

Btw, it also works with the gpt-4 model settings, now. Although I suppose that there is a fallback to the default gpt-3.5-turbo model? 🤔 (As I don't have api access to gpt-4, yet...)

**Comment by genieai-info** [comment]
@dahelg thanks for confirming. You likely have gpt-4 access already- it would fail if you didn't have API access yet. Which ChatGPT extension was that? We can try to reproduce the problem better.

**Comment by genieai-info** [comment]
@dahelg the editor view is defaulted to `text-davinci-003` to get the code response better. The configured model in your settings doesn't have an impact on the Editor View today.

What do you mean by "Conversation view does not work anyway"? Do you mean the keyboard shortcut doesn't work due to shortcut clash, or you can't use the Conversation View feature at all?

**Comment by theboy181** [comment]
Im getting a 404 error after putting in my API code

**Comment by Msameim181** [comment]
After changing the API Key, It does not work for me. No conversation, No editor. And there is no console log relevant to this issue.
